src/test/java/com/ot/restaurant/entities/IoCContainerTest.java

```diff
@@ -0,0 +1,85 @@
+package com.ot.restaurant.entities;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Objects;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class IoCContainerTest {
+
+    @Test
+    void singletonIoCContainer() {
+
+        IoCContainer ioc = IoCContainer.getInstance();
+        IoCContainer ioc2 = IoCContainer.getInstance();
+
+        assertEquals(ioc, ioc2);
+    }
```

Comment:
You don't need this test. Focus on testing what the code should do regarding your business logic

Reply:
done